tt_eager/tt_lib/csrc/tt_lib_bindings_tensor_dm_ops.cpp

```diff
@@ -292,15 +292,15 @@ namespace tt::tt_metal::detail{
 
         m_tensor.def("unpad", &unpad,
             py::arg("input").noconvert(), py::arg("output_tensor_start"), py::arg("output_tensor_end"), py::arg("output_mem_config").noconvert() = operation::DEFAULT_OUTPUT_MEMORY_CONFIG, R"doc(
-            Unpad TT Tensor.
+            Unpad TT Tensor. Also known as slice.
 
             Returns an output tensor from output tensor start indices ``arg1`` to output tensor end indices ``arg2`` (inclusive) of the input tensor.
 
             .. csv-table::
                 :header: "Argument", "Description", "Data type", "Valid range", "Required"
 
-                "input", "Input tensor", "Tensor", "", "Yes"
-                "output_tensor_start", "Start indices of input tensor", "List[int[4]]", "Must be all 0s", "Yes"
+                "input", "Input tensor", "Tensor", "List[int[4]]", "Yes"
+                "output_tensor_start", "Start indices of input tensor", "List[int[4]]", "", "Yes"
```

Comment:
Unpad also supports any tensor rank right? So doesn't need to specifically be 4?

Reply:
ah ya good point! fixed that :)